Sources/ContainerBuilder/ContainerBuilder+PullImage.swift

```diff
@@ -6,44 +6,32 @@ import Shell
 extension ContainerImageSpec {
```

Comment:
nothing to change here, but just wondering: all of the methods here are static. is that by design or just shortcut to get a working version out?